lib/process-services-cloud/src/lib/app/components/app-details-cloud.component.spec.ts

```diff
@@ -59,17 +56,17 @@ describe('AppDetailsCloudComponent', () => {
         component.applicationInstance = fakeApplicationInstance[2];
         fixture.detectChanges();
 
-        const theme = fixture.nativeElement.querySelector('.adf-app-listgrid-item-card').getAttribute('ng-reflect-ng-class');
-        const icon = fixture.nativeElement.querySelector('.adf-app-listgrid-item-card-logo-icon');
+        const card = host.querySelector('.adf-app-listgrid-item-card') as HTMLElement;
+        expect(card.classList.contains(DEFAULT_APP_INSTANCE_THEME));
 
-        expect(theme).toEqual(DEFAULT_APP_INSTANCE_THEME);
+        const icon = host.querySelector('.adf-app-listgrid-item-card-logo-icon');
         expect(icon).toBeTruthy();
     });
 
     it('should render card with a non ApplicationInstanceModel input object', () => {
-        component.applicationInstance =  { name: 'application-new-3', createdAt: '2018-09-21T12:31:39.000Z', status: 'Pending' };
+        component.applicationInstance = { name: 'application-new-3', createdAt: '2018-09-21T12:31:39.000Z', status: 'Pending' };
         fixture.detectChanges();
-        const app = fixture.nativeElement.querySelector('.mat-card');
+        const app = host.querySelector('.adf-app-listgrid-item-card');
```

Comment:
can we move host.querySelector('.adf-app-listgrid-item-card'); to reusable function to don't repeat whole selector each time?